<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KILL_Design</class>
 <widget class="QWidget" name="KILL_Design">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Nháº­p ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Kill_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>KILL</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
